src/shared_modules/utils/linuxInfoHelper.h

```diff
@@ -0,0 +1,62 @@
+/*
+ * Wazuh shared modules utils
+ * Copyright (C) 2015-2021, Wazuh Inc.
+ * December 10, 2021.
+ *
+ * This program is free software; you can redistribute it
+ * and/or modify it under the terms of the GNU General Public
+ * License (version 2) as published by the FSF - Free Software
+ * Foundation.
+ */
+
+#ifndef LINUXINFO_HELPER_H
+#define LINUXINFO_HELPER_H
+
+#include "filesystemHelper.h"
+#include <vector>
+#include <cstdint>
+#include <string>
+#include <unistd.h>
+
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wunused-function"
+
+namespace Utils
+{
+    static uint64_t getBootTime(void)
+    {
+        static uint64_t btime;
+
+        try
+        {
+            if (0UL == btime)
+            {
+                const std::string key {"btime "};
+                const auto file { Utils::getFileContent("/proc/stat") };
+
+                btime = std::stoull(file.substr(file.find(key) + key.length()));
+            }
+        }
+        // LCOV_EXCL_START
+        catch (...)
+        {}
+
+        // LCOV_EXCL_STOP
+
+        return btime;
+    }
+
+    static uint64_t getClockTick(void)
```

Comment:
For better readability, we could add comments describing what the value returned is, what physical dimensions it is in, etc.